src/pages/App.tsx

```diff
@@ -84,11 +88,40 @@ const Marginer = styled.div`
 `
 
 export default function App() {
-  useActiveNetwork()
-  const { account } = useActiveWeb3React()
+  const { account, chainId } = useActiveWeb3React()
   const aboutPage = useRouteMatch('/about')
   const apolloClient = useExchangeClient()
-
+  const dispatch = useDispatch<AppDispatch>()
+  const fetchGas = (chain: string) => {
+    fetch(`https://staging-krystal-api.knstats.com/${chain}/v2/swap/gasPrice`)
```

Comment:
should we move this to env like: `REACT_APP_KRYSTAL_API = https://staging-krystal-api.knstats.com`

Reply:
agree